core/tabs/system-setup/tab_data.toml

```
@@ -15,6 +15,12 @@ description = "This command installs a minimal arch server setup under 5 minutes
 script = "arch/server-setup.sh"
 task_list = "SI D"
 
+[[data.entries]]
+name ="Linux Neptune for SteamDeck"
+description = "Valve's fork of Linux Kernel for the SteamDeck"
+script = "arch/linux-neptune.sh"
+task_list = "I PFM K"
+
```

```suggestion

[[data.preconditions]]
matches = false
data = { file = "/sys/devices/virtual/dmi/id/board_vendor" }
values = ["Valve", "Jupiter"]

```
source: https://www.reddit.com/r/SteamDeck/comments/telfda/how_to_detect_if_software_is_running_on_steamdeck/

---

Okayy. Thank you. Will check with the issue's author to see if this works.